scanner/src/main/kotlin/Main.kt

```diff
@@ -91,13 +91,18 @@ object Main {
             order = 0)
     private var inputPath: File? = null
 
-    @Parameter(description = "The output directory to store the scan results and source code of packages.",
+    @Parameter(description = "The output directory to store the scan results in.",
             names = arrayOf("--output-dir", "-o"),
             required = true,
             order = 0)
     @Suppress("LateinitUsage")
     private lateinit var outputDir: File
 
+    @Parameter(description = "The output directory for downloaded source code. Defaults to output-dir/download.",
```

Comment:
I believe we should mark `output-dir` in here somehow to emphasize it's not meant to be literal, like `<output-dir>`.

Also, what about naming the sub-directory `downloader` (like the tool) or `downloads` (plural) instead of `download`?

Reply:
Done. I liked "downloads" more.